app/views/projects/user_show.html.haml

```diff
@@ -29,7 +29,8 @@
             %p
               %img{:src => "/usercover.jpg", :title => "Project Cover"}
               = link_to project.name, project.urlbase
-              = render partial: 'forked_from', locals: {project: project}
+              - @project = project
+              = render 'forked_from', :project => @project
```

Comment:
In **most** case we have the instance variable available, but there are a few where we don't. Like the case where we are iterating through the list of projects:
```Ruby
%div
   - @projects.each do |project|
     %div
       %p
```
Pipes don't support instance variable. 
Now, if we always pass it as local then first it is passed to project_header and then it project_header passes it to  fork_from, seems kinda ehh.. idk! Doesn't feel right.
But at least it will maintain uniformity through out the code.